src/main/java/com/commercetools/sync/categories/CategorySync.java

```diff
@@ -184,14 +185,14 @@ public CategorySync(@Nonnull final CategorySyncOptions syncOptions) {
                         return CompletableFuture.completedFuture(null);
                     }
 
-                    prepareDraftsForProcessing(categoryDrafts, keyToIdCache);
```

Comment:
javadocs are not up to date... check keyToIdCaches in code.

Reply:
94a6821d